.github/workflows/test.yml

```
@@ -39,41 +39,50 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - python: "3.7"
           - python: "3.8"
             backend: "etcd"
           - python: "3.9"
             backend: "consul"
           - python: "3.10"
           - python: "3.11"
+          - python: "3.12"
+          - python: "3.12"
+            traefik-version: v3.0.0-beta5
```

This adds two test runs when only one is really needed. We can get the same coverage if we add the v3 test to 3.11 and only add one 3.12 run:

```suggestion
            traefik-version: v3.0.0-beta5
          - python: "3.12"
```

---

Good point, I'll iterate a bit!